### PR TITLE
A beer can and a wine can

### PR DESCRIPTION
**Pull request by RumiTiger**
## About the PR
A beer can and a wine can have been added to the game, they can be found in boozeomat. Perhaps later I will add more alcoholic beverages in cans and a special vending machine for them.

## Why / Balance
Why didn't we have cans of beer? We need to fix this urgently!

## Media
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
photo_2024-01-22_19-23-10

**Changelog**
🆑
• add: A beer can and a wine can have been added to the game, they can be found in boozeomat.

### CODE REVIEW

**Comment by Jackal298** [comment]
add a bottle of sake, there is a drink in the game but there is no legal way to get it

**Comment by UbaserB** [comment]
> add a bottle of sake, there is a drink in the game but there is no legal way to get it

No, you can get it from juicing lemoons.

**Comment by Jackal298** [comment]
> > add a bottle of sake, there is a drink in the game but there is no legal way to get it
> 
> No, you can get it from juicing lemoons.

How can you make a Japanese rice-based alcoholic drink from lemons?

**Comment by lzk228** [comment]
sprites are mid

**Comment by RumiTiger** [comment]
> спрайты средние

Perhaps the very idea of ​​writing letters on cans instead of drawings is not very good

**Comment by Aisu9** [comment]
> > > add a bottle of sake, there is a drink in the game but there is no legal way to get it
> > 
> > 
> > No, you can get it from juicing lemoons.
> 
> How can you make a Japanese rice-based alcoholic drink from lemons?

lemoons... mutated lemon

**Comment by K-Dynamic** [comment]
The sprites look like a combination of generic branding and craft brewery labels

**Comment by potato1234x** [comment]
cans with letters should be placeholders

**Comment by RumiTiger** [comment]
> Happy to contribute a beer and wine can sprite (loosely inspired by a few real-world designs) if you like:
> 
> beer wine
> 
> Edit: zoomed-in versions: image image

Wow, they're very good!

**Comment by potato1234x** [comment]
Why close :despair:

**Comment by RumiTiger** [comment]
> Why close :despair:

I opened it again

File: Resources/Textures/Objects/Consumable/Drinks/beer_can.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "The sprite made by DrMelon and the code wrote by RumiTiger",
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Consumable/Drinks/beer_can.rsi/meta.json:8]:
```suggestion
    "copyright": "Github #DrMelon",
```

**Pull request merged**